src/olympia/files/tests/test_models.py

```diff
@@ -425,10 +425,13 @@ def setUp(self):
     def parse(self, addon=None, filename='extension.xpi', minimal=None):
         path = 'src/olympia/files/fixtures/files/' + filename
         xpi = os.path.join(settings.ROOT, path)
-        if minimal is None:
-            return parse_addon(open(xpi), addon)
-        else:
-            return parse_addon(open(xpi), addon, minimal=minimal)
+        kwargs = {}
+
+        if minimal is not None:
+            kwargs = {'minimal': minmal}
```

Comment:
typo (let travis finish before fixing it though, it will reveal whether or not it's covered by tests correctly...)

Reply:
whow, good catch! Just saw that error in a local flake8 run though, haha. I'm too quick to assign someone for review.

Reply:
pushed a fix, it got caught during tests :)